spec/models/education_spec.rb

```diff
@@ -1,5 +1,58 @@
 require 'rails_helper'
 
 RSpec.describe Education, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  
+  it "has a valid factory" do
+    expect(FactoryGirl.build(:education)).to be_valid
+  end
+
+  describe "converts date to strftime" do
+    education = FactoryGirl.build(:education)
+    it 'should convert education start date to strftime' do
```

Comment:
So the `should` syntax is outdated. Use something like `it 'converts education start date to strftime'`. Be sure to change the other tests as well in here